code/modules/migrations/migrations_controller.dm

```diff
@@ -0,0 +1,114 @@
+/datum/migration_controller
```

Comment:
[Please document your code to meet dmdoc requirements.](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)